### PR TITLE
Hide outputs page on cloud

### PR DESCRIPTION
**Pull request by Gaya**
Users on cloud shouldn't be able to setup outputs

## Description

Hides the system outputs page for cloud users

Please not that this includes #9423, so please wait until this is merged

## Motivation and Context

Users on cloud shouldn't be able to setup outputs

## How Has This Been Tested?

Checking output of pages and menu

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by Gaya** [comment]
Fixes https://github.com/Graylog2/graylog-plugin-cloud/issues/90

**Comment by edmundoa** [comment]
I noticed that this PR only takes care of System Outputs (`/system/outputs`), but we should also remove Stream Outputs (`/streams/<stream_id>/outputs`), since that would also let users send messages out of Graylog Cloud. @Gaya do we want to also hide that in this PR or better create a new one for it?

**Comment by Gaya** [comment]
I'll add it to this one. Thanks for noticing and letting me know

**Comment by edmundoa** [comment]
I just realised that there's another thing we should hide in the Streams page. It is the Index set a Stream should use. Since it's only a form field and we have modified some things on that page here, I think we can also hide it here, @Gaya what do you think?
Screen Recording 2020-11-16 at 14 11 22

**Comment by Gaya** [comment]
@edmundoa sure, it's going to "default" anyway so I guess that's enough for now. I'll hide the select

File: graylog2-web-interface/src/components/streams/StreamForm.jsx [hunk 1]
```diff
@@ -84,14 +85,15 @@ class StreamForm extends React.Component {
     this.setState(change);
   };
 
-  state = this._getValuesFromProps(this.props);
+  _indexSetSelect = () => {
+    const { indexSetId } = this.state;
 
-  render() {
-    const { title, description, removeMatchesFromDefaultStream, indexSetId } = this.state;
+    if (!AppConfig.isCloud()) {
```
Comment (edmundoa) [inline graylog2-web-interface/src/components/streams/StreamForm.jsx:91]: It should be the other way around, or? :)

```suggestion
    if (AppConfig.isCloud()) {
```

Comment (Gaya) [reply graylog2-web-interface/src/components/streams/StreamForm.jsx:91]: Hah! I used that to test the opposite. Good catch!

**Review by edmundoa — APPROVED** [review]
Looks good to me :shipit:

**Pull request merged**